spiped/README

```diff
@@ -21,9 +21,12 @@ Options:
 		[ipv6::addr]:port
 	Note that hostnames are resolved when spiped is launched and are not
 	re-resolved later; thus if DNS entries change spiped will continue to
-	connect to the expired address.
+	accept at the expired address.
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
```

Comment:
> My first instinct is to remove the /*/README files since that info seems to be the the man-pages

I agree on this. If the idea of my changes is fine I'll fix the stylistic and grammatical changes, remove the redundant `README` files and update this pull request?